### PR TITLE
Update select command

### PR DESCRIPTION
**Pull request by robinloh**

### CODE REVIEW

**Comment by chenzhiliang94** [comment]
eh you still have System.out.println lol

**Comment by robinloh** [comment]
WIP - Working on testing UI for select command

**Comment by waterhall** [comment]
the missed lines damn easy to cover

**Comment by robinloh** [comment]
I hope i have covered the test cases...

**Review by chenzhiliang94 — COMMENTED** [review]
Eh for this. I'm not sure about this sia. I thought usually test case is input a command followed by assertion? This kind of assert are they allowed? @e0003133 @JTLX

File: src/test/java/seedu/ezdo/model/task/TaskTest.java [hunk 1]
```diff
@@ -21,4 +21,12 @@ public void hashCode_isEqual() throws Exception {
         assertTrue(task.hashCode() == task.hashCode());
     }
 
+    @Test
```
Comment (chenzhiliang94) [inline src/test/java/seedu/ezdo/model/task/TaskTest.java]: @e0003133 @JTLX eh I not sure if tests can be done like this? I thought always need to input a command followed by assertion?

**Review by jtlx — APPROVED** [review]

**Pull request merged**